pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -22,15 +24,21 @@ type EtcdMembersController struct {
 	etcdClient     etcdcli.EtcdClient
 }
 
-func NewEtcdMembersController(operatorClient v1helpers.OperatorClient,
+func NewEtcdMembersController(
+	livenessChecker *health.MultiAlivenessChecker,
+	operatorClient v1helpers.OperatorClient,
 	etcdClient etcdcli.EtcdClient,
 	eventRecorder events.Recorder,
 ) factory.Controller {
 	c := &EtcdMembersController{
 		operatorClient: operatorClient,
 		etcdClient:     etcdClient,
 	}
-	return factory.New().ResyncEvery(time.Minute).WithSync(c.sync).ToController("EtcdMembersController", eventRecorder.WithComponentSuffix("member-observer-controller"))
+
+	syncer := health.NewCheckingSyncWrapper(c.sync, 5*time.Minute)
```

Comment:
Nit: Seems like every controller is using 5 mins as the default threshold. Thoughts on passing this as an arg to all controllers in starter.go? That way we don't have to change it everywhere if we need to tweak this again.

Reply:
I wanted to have this next to the "ResyncEvery" code, but I can add a default value. The only controller that doesn't have the 5m set is the defrag controller.